appinventor/aicompanionapp/src/Base.lproj/Main.storyboard

```diff
@@ -152,10 +186,15 @@
     </scenes>
     <resources>
         <image name="Instructions.png" width="800" height="200"/>
-        <image name="connectwCode.png" width="300" height="49"/>
-        <image name="connectwQR.png" width="300" height="49"/>
+        <image name="Mooning" width="400" height="400"/>
```

Comment:
this is actually not in the codebase any longer. Beyond perplexed why it is showing up